### DISCUSSION TITLE
Is this project compatible with Jetson 2GB $59 model

### DISCUSSION THREAD

**Discussion by kodalivk** [NVIDIA-AI-IOT/jetbot#362] [Q&A] [answered]
Hello - I am trying to assemble the jetbot and see that this project is based on the original jetson nano, but wanted to make sure if this will work for the Jetson 2GB model as well. I know that the WiFi module is USB based in the new jetson, but other than that are there any challenges that I might encounter? Thanks for your response.

**Comment by jaybdub** [comment, marked as answer]
Hey @kodalivk ,

Thanks for reaching out!

Yes, JetBot is compatible with Jetson Nano 2GB.  However, for best performance With reduced memory it requires slightly a different configuration, which we have done in the released SD card images.  

Please let me know if this helps or you have any questions.

Best,
John

**Comment by jaybdub** [comment]
FYI, You can find the 2GB software setup instructions here

http://jetbot.org/master/software_setup/sd_card.html

Best,
John
